pyface/api.py

```diff
@@ -28,13 +28,16 @@
 from .key_pressed_event import KeyPressedEvent
 from .message_dialog import error, information, warning, MessageDialog
 from .progress_dialog import ProgressDialog
-try:
+
+from .util._optional_dependencies import optional_import as _optional_import
+
+# Excuse pygments dependency (for Qt), otherwise re-raise
+with _optional_import("pygments", msg="PythonEditor not available"):
```

Comment:
The log message doesn't mention that the reason it isn't available is because Pygments isn't installed - that would be useful to know.